src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/esa/EsaIssueFinalDecisionAboutToSubmitHandlerTest.java

```diff
@@ -244,6 +243,15 @@ public void givenAnIssueFinalDecisionEventForGenerateNoticeFlowWhenAllowedOrRefu
         callback.getCaseDetails().getCaseData().getSscsFinalDecisionCaseData().setWriteFinalDecisionAllowedOrRefused("allowed");
         callback.getCaseDetails().getCaseData().getSscsFinalDecisionCaseData().setWriteFinalDecisionGenerateNotice("yes");
 
+        Hearing hearing = Hearing.builder().value(HearingDetails.builder()
```

Comment:
```suggestion
        HearingDetails details = HearingDetails.builder()...
        Hearing hearing = Hearing.builder().value(details)
```
Basically, separate out any nested builders, easier to read. Could extend to other test classes you've updated too.